torch_xla/csrc/xla_lower_util.cpp

```diff
@@ -72,6 +73,43 @@ std::pair<xla::XlaOp, xla::XlaOp> DotBroadcast(const xla::XlaOp& lhs,
 
 }  // namespace
 
+std::vector<xla::XlaOp> CreateKthValue(const xla::XlaOp& input, xla::int64 k,
+                                       xla::int64 dim, bool keepdim) {
+  // Here 'k' is 1 based (1...).
+  xla::Shape shape = XlaHelpers::ShapeOfXlaOp(input);
+  XLA_CHECK_LE(k, shape.dimensions(dim));
+  xla::Shape iota_shape =
+      xla::ShapeUtil::MakeShape(xla::PrimitiveType::S32, shape.dimensions());
+  xla::XlaOp iota = xla::Iota(input.builder(), iota_shape, dim);
+  // TODO: Remember to add is_stable=true as last Sort() argument when fetching
```

Comment:
Why would we need stability here?

Reply:
To return indices in a predictable fashion.
Though, I am not sure what the PT semantics are on ties, so it might not matter.

Reply:
The documentation doesn't say anything about it, I'd rather not do it. Stable sort is slower.

Reply:
Sure.